en/contribute/support.md

```diff
@@ -9,17 +9,13 @@ This section shows how you can get help from the core dev team and the wider com
 
 ## Forums and Chat
 
-The core development team and community are active on the following channels.
+The core development team and community are active on the following channels:
```

Comment:
Note, the logos are not particularly useful - in particular the PX4 logo doesn't tell you "discuss forum". Words are better here.